sections/section7_comparative_analysis.md

```diff
@@ -11,6 +11,9 @@ Norwegian Singles employs a consistent year-round approach with sub-threshold as
 ## vs. Pfitzinger Method
 Norwegian Singles distributes quality evenly through the week (E-Q-E-Q-E-Q-LR) focusing primarily on sub-threshold work, while Pfitzinger incorporates medium-long midweek runs and uses more varied intensities (including VO2max and specific race-pace work). Pfitzinger's plans are explicitly designed with marathon-specific elements and feature more complex mesocycle periodization rather than Norwegian Singles' consistent weekly pattern.
 
+## vs. Palladino Power Project
+Norwegian Singles are strictly sub-threshold intensities with very short recoveries, while Steve Palladino incorporates supra-threshold and V̇O₂ max workouts into his training plans. The recoveries between intervals in Palladino's plans are also notably longer, averaging three minutes. Palladino plans have integrated CP testing every four weeks while the athlete running Nowegian Singles would need to integrate regular CP testing on their own. All of Palladino's plans are centered around power, not pace or HR, so if an athelte's CP is valid, Norwegian Singles can be executed accurately without the need of a lactate meter.
```

Comment:
Could you add small definition of CP between brackets? Something like "Palladino plans have integrated CP (Critical Power) testing ..."

Reply:
Yes. Also, I think I'll rewrite the last sentence so it flows a bit better with the topic.